march_hardware/src/PDOmap.cpp

```diff
@@ -8,124 +8,143 @@ namespace march4cpp
 {
 PDOmap::PDOmap()
 {
-  this->initAllObjects();
+  this->allObjects[IMCObjectName::StatusWord] = IMCObject(0x6041, 16);
+  this->allObjects[IMCObjectName::ActualPosition] = IMCObject(0x6064, 32);
+  this->allObjects[IMCObjectName::MotionErrorRegister] = IMCObject(0x2000, 16);
+  this->allObjects[IMCObjectName::DetailedErrorRegister] = IMCObject(0x2002, 16);
+  this->allObjects[IMCObjectName::DCLinkVoltage] = IMCObject(0x2055, 16);
+  this->allObjects[IMCObjectName::DriveTemperature] = IMCObject(0x2058, 16);
+  this->allObjects[IMCObjectName::ActualTorque] = IMCObject(0x6077, 16);
+  this->allObjects[IMCObjectName::CurrentLimit] = IMCObject(0x207F, 16);
+  this->allObjects[IMCObjectName::MotorPosition] = IMCObject(0x2088, 32);
+  this->allObjects[IMCObjectName::ControlWord] = IMCObject(0x6040, 16);
+  this->allObjects[IMCObjectName::TargetPosition] = IMCObject(0x607A, 32);
+  this->allObjects[IMCObjectName::TargetTorque] = IMCObject(0x6071, 16);
+  this->allObjects[IMCObjectName::QuickStopDeceleration] = IMCObject(0x6085, 32);
+  this->allObjects[IMCObjectName::QuickStopOption] = IMCObject(0x605A, 16);
 }
 
-void PDOmap::addObject(IMCObjectName objectname)
+void PDOmap::addObject(IMCObjectName objectName)
```

Comment:
Actually parameters should be **under_score** (https://wiki.ros.org/CppStyleGuide#Function_.2BAC8_Methods)

Reply:
changed all the class variables in the PDO-map files to match the cpp style guide